packages/development/vortex/development/train.py

```diff
@@ -13,7 +13,8 @@ def main(args):
 
     ckpt_in_cfg = 'checkpoint' in config
     if ckpt_in_cfg:
-        config.checkpoint = config.checkpoint.rstrip(',')
+        if config.checkpoint is not None:
```

Comment:
rather than adding more if, you could just directly add the check in the `ckpt_in_cfg`, e.g.:
```
ckpt_in_cfg = 'checkpoint' in config and config.checkpoint is not None
```

Reply:
good idea, thanks